app.py

```diff
@@ -152,23 +198,37 @@ def parse():
         'server_error': response.get('Error'),
         'version': version(),
         'username': username
-    })
+    }
+
+    if os.getenv ('REDIRECT_AB_TEST') == os.getenv ('HEROKU_APP_NAME') and os.getenv ('HEROKU_APP_NAME'):
```

Comment:
How do you see this being used?

Does the `REDIRECT_AB_TEST` variable need to be set on BOTH `hedy-beta` and `hedy-test` ?

Feels like an `HEDY_IS_TEST_ENV` variable might be more appropriate.